modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -1126,6 +1128,196 @@ public void testEnableAccountEntryRestricted() throws Exception {
 		}
 	}
 
+	@Test
+	public void testEnableAccountEntryRestrictedForNondefaultStorageType()
```

Comment:
It is not usual creating new methods for a new test in Objects. What the time usually does is to add the new test (block of code) inside an existing test. So, you add this new test (block of code) after a comment (as you did inside this method).